identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -12,7 +12,11 @@
                 "email-subscription--subscribed" -> subscription.subscribedTo,
                 "email-subscription--first" -> row.isFirst))">
                 @if(subscription.subscribedTo){<input type="hidden" name="emailSubscription[]" value="@subscription.listId" />}
-                <div class="email-subscription__name">@subscription.name</div>
+                <div class="email-subscription__name">@subscription.name
+                    @if(subscription.subheading.isDefined){
+                        <div class="email-subscription__name">@subscription.subheading</div>
```

Comment:
Thank you, that looks much nicer! I'm keeping the div as a wrapper to make a 'column' (to the left of the description) for the name and subheading:

<img width="815" alt="picture 260" src="https://cloud.githubusercontent.com/assets/8607683/18907789/edd57f04-8564-11e6-99ca-664b443a9f82.png">